src/main/java/de/butbetter/keksys/Calculator.java

```diff
@@ -0,0 +1,5 @@
+package de.butbetter.keksys;
+
+public interface Calculator {
+	Float calculateVAT(float input, float percent);
```

Comment:
Why *float/Float* and not *double/Double*?

*input* is a bad name, would rather take *price* or *priceWithoutVAT*